packages/wrangler/src/dev/remote.tsx

```diff
@@ -88,29 +90,6 @@ export function Remote(props: RemoteProps) {
 		port: props.port,
 	});
 
-	usePreviewServer({
-		previewToken,
-		assetDirectory: props.isWorkersSite
-			? undefined
-			: props.assetPaths?.assetDirectory,
-		localProtocol: props.localProtocol,
-		localPort: props.port,
-		ip: props.ip,
-	});
-
-	useInspector({
-		inspectorUrl:
-			props.inspect && previewToken
-				? previewToken.inspectorUrl.href
-				: undefined,
-		port: props.inspectorPort,
-		logToTerminal: true,
-		sourceMapPath: props.sourceMapPath,
-		host: previewToken?.host,
-		name: props.name,
-		sourceMapMetadata: props.bundle?.sourceMapMetadata,
-	});
-
```

Comment:
No more preview server or inspector server, replaced by ProxyWorker and ProxyInspectorWorker respectively